documentation/asciidoc/topics/con_caches.adoc

```diff
@@ -13,3 +13,17 @@ To create caches with {brandname} clusters running on {k8s}, you should:
 * Access {brandname} Console and create caches in XML or JSON format.
 
 You can use Hot Rod or HTTP clients but {brandname} recommends `Cache` CR or `Batch` CR unless your specific use case requires programmatic remote cache creation.
+
+[discrete]
+== Cache CRs
+
+When using `Cache` CRs, the following rules apply:
+
+* `Cache` CRs apply to {datagridservice} pods only.
+* You can create a single cache for each `Cache` CR.
+* If your `Cache` CR contains a template and a cache configuration, {ispn_operator} uses the template.
```

Comment:
This line might be confusing. Cache CR contains `template` and `templateName`, where the first should be cache configuration and the second one refers to an existing `template` on the server. I blame poor choice of the Cache CR fields in here.

Reply:
What do you think if we drop this as well as the procedure for creating caches from templates? I think we're moving away from the cache templates and they don't serve much purpose. I can't see anyone creating a cache from a template via the CR. Maybe from the console for a demo or something.